src/lib/containers/synapse_table_functions/SynapseTableCell.tsx

```diff
@@ -77,150 +46,152 @@ export const SynapseTableCell: React.FC<SynapseTableCellProps> = ({
-        />
+        <>
+          <DirectDownload
+            associatedObjectId={tableEntityId!}
+            associatedObjectType={FileHandleAssociateType.TableEntity}
+            fileHandleId={columnValue}
+            displayFileName={true}
+          />
+        </>
       )
+    case ColumnType.ENTITYID_LIST: {
+      const jsonData: string[] = JSON.parse(columnValue)
+      return <EntityIdList entityIdList={jsonData} />
     }
-  } else {
-    const isShort = isShortString(columnValue)
-    if (isShort) {
-      return <p className={isBold}> {columnValue}</p>
-    } else {
+    // handle other list types
+    case ColumnType.STRING_LIST:
+    case ColumnType.INTEGER_LIST:
+    case ColumnType.USERID_LIST: {
+      const jsonData: string[] = JSON.parse(columnValue)
       return (
-        <div className={isBold}>
-          <ElementWithTooltip
-            tooltipText={columnValue}
-            callbackFn={noop}
-            idForToolTip={`${colIndex}_${rowIndex}`}
-          >
-            <p className={isBold}> {columnValue}</p>
-          </ElementWithTooltip>
-        </div>
+        <p>
+          {jsonData.map((val: string, index: number) => {
+            return (
+              <span key={val} className={isBold}>
+                {val}
+                {index !== jsonData.length - 1 ? ', ' : ''}
+              </span>
+            )
+          })}
+        </p>
       )
     }
+    case ColumnType.DATE:
+      return (
+        <p className={isBold}>
+          {new Date(Number(columnValue)).toLocaleString()}
+        </p>
+      )
+
+    case ColumnType.USERID:
+      if (
+        Object.prototype.hasOwnProperty.call(mapUserIdToHeader, columnValue)
+      ) {
+        const { ownerId, userName } = mapUserIdToHeader[columnValue]
+        if (mapUserIdToHeader[columnValue].isIndividual === false) {
+          // isUserGroupHeader
+          const icon =
+            userName === AUTHENTICATED_USERS ? 'globe-americas' : 'users'
+          if (userName === AUTHENTICATED_USERS) {
+            return (
+              <span>
+                <FontAwesomeIcon icon={icon} /> All registered Synapse users
+              </span>
+            )
+          }
+          return (
+            <a
+              target="_blank"
+              rel="noopener noreferrer"
+              href={`${PRODUCTION_ENDPOINT_CONFIG.PORTAL}#!Team:${ownerId}`}
+            >
+              <FontAwesomeIcon icon={icon} /> {userName}
+            </a>
+          )
+        } else {
+          // isUserCard
+          return (
+            <UserCard
+              userProfile={mapUserIdToHeader[columnValue]}
+              preSignedURL={mapUserIdToHeader[columnValue].clientPreSignedURL}
+              size={'SMALL USER CARD'}
+            />
+          )
+        }
+      }
+      break
+    case ColumnType.STRING:
+    case ColumnType.DOUBLE:
+    case ColumnType.INTEGER:
+    case ColumnType.BOOLEAN:
+    case ColumnType.LINK:
```

Comment:
I need to figure out what a link column is. Sounds like it should be handled differently.